src/sentry/models/user.py

```diff
@@ -106,6 +111,30 @@ def get_avatar_type(self):
             return avatar.get_avatar_type_display()
         return 'letter_avatar'
 
+    def send_confirm_emails(self, is_new_user=False):
+        from sentry import options
+        from sentry.utils.email import MessageBuilder
+
+        for email in self.emails.filter(is_verified=False):
+            if not email.hash_is_valid():
+                email.set_hash()
+                email.save()
+
+            context = {
+                'user': self,
+                'url': absolute_uri(reverse(
+                    'sentry-account-confirm-email',
+                    args=[self.id, email.validation_hash]
+                )),
+                'is_new_user': is_new_user,
+            }
+            msg = MessageBuilder(
+                subject='%sConfirm Email' % (options.get('mail.subject-prefix'),),
+                template='sentry/emails/confirm_email.txt',
```

Comment:
Add `type='user.confirm_email',` below this line to get sweet sweet `mail.{queued,sent}` logging events.

If you run tests and they wig out or something doesn't work, rebasing off of `origin/master` will fix it.